csrc/backend_ops/tensorrt/batched_rotated_nms/trt_batched_rotated_nms.cpp

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) OpenMMLab. All rights reserved
```

Comment:
may  update the doc for trt ops: https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/ops/tensorrt.md